test/reducers/reducer-test.js

```diff
@@ -1,6 +1,7 @@
 import expect from 'expect'
 import todos from '../../src/reducers/todos'
 import components from '../../src/reducers/components'
+import { CP_TextBox, CP_Button, CP_Dropdown, CP_TextArea, CP_Link, CP_Radio, CP_ButtonGroup} from '../../src/components/ToolBoxComponents';
```

Comment:
If you feel like the list of imports is getting too long you have two options:

1. In addition to exporting each of these objects independently from a module, add them all to a single object and export it.
1. Use a wildcard (e.g. `import * as ToolBoxActions from '../../src/components/ToolBoxComponents';`